components/radio/style/index.less

```diff
@@ -12,6 +12,7 @@
   .reset-component;
 
   display: inline-block;
+  font-size: 0;
```

Comment:
`font-size: 0;` here results in the following - https://prnt.sc/u2sqc7 - that we set it `14px` and then back to `0` for `.ant-radio-group` class. This seems like a bug. Because If I render some custom content as a child of `RadioGroup` component it's font-size is 0.

@xrkffgg 
What do we need to have this fixed? to create an issue or to share some link with reproduction here?